js/main.js

```diff
@@ -110,6 +111,116 @@ var Menu = {
   },
 };
 
+// AJAX
+Ajaxy = {
+  init: function() {
+    var _this = this;
+    
+    var siteURL = "http://" + top.location.host.toString();
+
+    _this.$ajaxyLinks = $("a[href^='" + siteURL + "'], a[href^='/'], a[href^='./'], a[href^='../'], a[href^='#']");
+    //_this.$elementsToHide = $('.nav, #main-container, #three-scene');
+
+    // Find all ajaxy links and bind ajax event
+    _this.$ajaxyLinks.click(function(event) {
+      event.preventDefault();
+
+      var url = event.currentTarget.href;
+
+      _this.ajaxLoad(url);
+
+    });
+
+    $(document).ready( function() {
+      // For back button
+      window.onpopstate = function() {
+        _this.ajaxLoad(document.location.origin + document.location.pathname);
+      };
+    });
+  },
+
+  reset: function() {
+    var _this = this;
+
+    // Unbind click from all ajax links
+    _this.$ajaxyLinks.unbind('click');
+
+    // Re initiate
+    _this.init();
+  },
+
+  ajaxLoad: function(url) {
+    var _this = this;
+
+    $.ajax(url, {
+      beforeSend: function() {
+        _this.ajaxBefore();
+      },
+
+      dataType: 'html',
+      error: function(jqXHR, textStatus) {
+        _this.ajaxErrorHandler(jqXHR, textStatus);
+      },
+
+      success: function(data) {
+        _this.ajaxSuccess(data, url);
+      },
+
+      complete: function() {
+        _this.ajaxAfter();
+      },
+    });
+  },
+
+  ajaxBefore: function() {
+    var _this = this;
+
+    $('body').addClass('loading');
+    $('body, html').animate({
+      scrollTop: 0,
+    }, fastAnimationSpeed);
+  },
+
+  ajaxAfter: function() {
+    var _this = this;
+
+    $('body').removeClass('loading');
+
```

Comment:
not going to block merge for this but think the Menu function to close the drawers other than radio should trigger here